src/content/release-notes/browser-rendering.yaml

```
@@ -5,6 +5,10 @@ productLink: "/browser-rendering/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+  - publish_date: "2025-04-30"
+    title: Automatic Request headers
```

For consistency with the description. 
```suggestion
    title: Automatic Request Headers
```